host/xtest/regression_6000.c

```diff
@@ -466,7 +466,7 @@ static TEEC_Result check_storage_available(uint32_t id, bool *avail)
 	switch (res) {
 	case TEEC_SUCCESS:
 		*avail = true;
-		fs_unlink(&sess, obj);
+		res = fs_unlink(&sess, obj);
```

Comment:
This should normally not fail if `fs_create()` has succeeded. How about logging the error before passing it on?

Reply:
I think some prominent logging or return value checks should be added here (return value checks may be more in line with code standards?). I discovered this omission because when I was using xtest on qemu to verify secure storage, fs_create succeeded, but fs_unlink failed, and xtest did not stop, so I spent more time finding the first error.

Reply:
I asked for logging because it might be hard to tell the unexpected error apart from an error returned by `fs_create()` above.

Reply:
> I asked for logging because it might be hard to tell the unexpected error apart from an error returned by `fs_create()` above.

I have add the logging for `fs_unlink` in the letest patch, thanks for your review.